src/test/java/tests/boardTests/listTests/MovingCardsBetweenLists.java

```diff
@@ -0,0 +1,61 @@
+package tests.boardTests.listTests;
+
+import api.clients.ApiBoardClient;
+import api.clients.ApiCardClient;
+import api.clients.ApiListClient;
+import api.pojo.requests.BoardBuilder;
+import api.pojo.requests.CardBuilder;
+import api.pojo.requests.ListBuilder;
+import com.codeborne.selenide.Selenide;
+import jdk.jfr.Description;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import tests.TestInit;
+
+public class MovingCardsBetweenLists extends TestInit {
+
+    private final SoftAssert softAssert = new SoftAssert();
+    private final BoardBuilder boardBody = BoardBuilder.builder().build();
+    private final ListBuilder listBuilder = ListBuilder.builder().build();
+    private final CardBuilder cardBuilder = CardBuilder.builder().build();
+    private final ApiListClient apiListClient = new ApiListClient(BASE_URL);
+    private final ApiBoardClient apiBoardClient = new ApiBoardClient(BASE_URL);
+    private final ApiCardClient apiCardClient = new ApiCardClient(BASE_URL);
+
+    private String boardId;
+    private String idListSource;
+    private String idListTarget;
+    private String idCardFromListSource;
+    private String idCardFromListTarget;
+
+
+    @BeforeMethod
+    public void setup() {
+        boardId = apiBoardClient.createNewBoard(boardBody, 200).getId();
```

Comment:
In the current TestInit, we create a board in DeforeMethod. Do you really need two boards?